x-pack/legacy/plugins/lens/public/indexpattern_plugin/filter_ratio.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ExpressionFunction } from 'src/legacy/core_plugins/interpreter/types';
+import { KibanaDatatable } from '../types';
+
+interface FilterRatioKey {
+  id: string;
+}
+
+export const calculateFilterRatio: ExpressionFunction<
+  'lens_calculate_filter_ratio',
+  KibanaDatatable,
+  FilterRatioKey,
+  KibanaDatatable
+> = {
+  name: 'lens_calculate_filter_ratio',
+  type: 'kibana_datatable',
+  help: i18n.translate('xpack.lens.functions.calculateFilterRatio.help', {
+    defaultMessage: 'A helper to collapse two filter ratio rows into a single row',
+  }),
+  args: {
+    id: {
+      types: ['string'],
+      help: i18n.translate('xpack.lens.functions.calculateFilterRatio.id.help', {
+        defaultMessage: 'The column ID which has the filter ratio',
+      }),
+    },
+  },
+  context: {
+    types: ['kibana_datatable'],
+  },
+  fn(data: KibanaDatatable, { id }: FilterRatioKey) {
+    const newRows: KibanaDatatable['rows'] = [];
+
+    if (data.rows.length === 0) {
+      return data;
+    }
+
+    const entries = Object.entries(data.rows[0]);
```

Comment:
If entries is only used on the next line, I don't see a lot of value in adding another named variable to the program. It's one more mental thing for a reader of the code to track.

Reply:
This was left over from a previous effort. Simplified.